g4f/Provider/deprecated/__init__.py

```diff
@@ -18,4 +18,8 @@
 from .Aibn          import Aibn
```

Comment:
Consider organizing the import statements alphabetically for better readability.